Sources/VitaminCore/Foundations/Icons/Vitamix.swift

```diff
@@ -409,8 +415,10 @@ public enum Vitamix {
       public static let play = VitaminImageAsset(name: "line/media/play")
       public static let playCircle = VitaminImageAsset(name: "line/media/play_circle")
       public static let `repeat` = VitaminImageAsset(name: "line/media/repeat")
+      public static let shoppingBag = VitaminImageAsset(name: "line/media/shopping_bag")
```

Comment:
Do we need this one? It's already in finance

Reply:
you're right, I think I made a wrong drag and drop, thought it had not work, and reimported the icon in right section, before running Swiftgen.
Fixed !